apps/feedback/models/response_comment.go

```diff
@@ -0,0 +1,11 @@
+package models
+
+import "github.com/jinzhu/gorm"
+
+type ResponseComment struct {
```

Comment:
How would I know who added that comment ? A Question response can have multiple comments..i.e. a conversation.

Reply:
refactored as discussed